### ISSUE TITLE
asNavFor mirror groupCell of it's parent

### ISSUE THREAD

**Issue by amdad** [metafizzy/flickity#419]
Hey David! @desandro 
Thanks for greatly awaited groupCell. Works flawless.

I tried to use it with David asNavFor and here is my feature request.

I used main flickity with auto groupCell, so it changes dynamically number of  is-selected (in viewarea) cells. And here is a problem. Navigation is-selected cells are not "chained" with those in main carousel. I know it have separate settings but none of them mirror parent carousel.

I don't want to change how navigation works. Because it's OK (my settings below). I just want to have is-selected active classes in nav to exactly highlight the same cells as it's target carousel. Already (in my settings) only one cell is-selected, I don't think it should be hard to add. Am I wrong? 

This is cosmetic thing just for highlight/tell user which cells are already displayed so they are no longer active as links to move forward/backward. Because number of slides according to them widths and viewarea are dynamic, so none of current "nav" settings match this goal.

```js
      prevNextButtons: false,
      pageDots: false,
      margin: 5,
      asNavFor: '.carousel',
      wrapAround: true, 
      cellAlign: 'center',
      lazyLoad: 8,
      freeScroll: true 
```

**Comment by desandro** [comment]
Thank you for reporting this issue. I've fixed this behavior in flickity-as-nav-for. This will be fixed in the next release.

**Comment by amdad** [comment]
Wahoo! Thanks David!

**Comment by amdad** [comment]
Included in 2.0.3!
